src/main/java/com/powsybl/openloadflow/network/impl/LfStaticVarCompensatorImpl.java

```diff
@@ -60,6 +63,9 @@ public double getMaxQ(double p) {
         if (svc.getRegulationMode() == StaticVarCompensator.RegulationMode.VOLTAGE) {
             setVoltageControl(svc.getVoltageSetpoint(), svc.getRegulatingTerminal(), breakers, report);
         }
+        if (svc.getExtension(VoltagePerReactivePowerControl.class) != null) {
```

Comment:
Indeed, as we have the parameter in `LfNetworkParameters`, why we don't use it here? If it is false, can we set the slope to zero?